src/patterns/components/inputs/collection.yaml

```diff
@@ -89,6 +89,12 @@ variableTable:
   $sprk-text-input-border-radius:
     default: 4px
     description: The border radius to use for text input elements.
+  $sprk-text-input-huge-border:
+    default: 1px solid $sprk-black-tint-25
+    description: The border around huge text input elements.
+  $sprk-text-input-huge-focus-border-width:
+    default: 2px
+    description: The border radius to use for huge text input elements when it's focused.
```

Comment:
Can't figure out how to mark the line in git but line 134 has a copy/paste error for the description of sprk-text-input-huge-font-size